svtools/bedpetobed12.py

```diff
@@ -191,7 +191,8 @@ def epilog():
     return 'The input BEDPE file may be gzipped. If the input file is omitted then input is read from stdin. Output is written to stdout.' 
 
 def add_arguments_to_parser(parser):
-    parser.add_argument('-b', '--bedpe', metavar='<BEDPE>', default=None, help='BEDPE input file')
+    parser.add_argument('-i', '--input', metavar='<BEDPE>', default=None, help='BEDPE input file')
+    parser.add_argument('-o', '--output', metavar='<BED12>', type=argparse.FileType('w'), default=sys.stdout, help='Output BED12 to write (default: stdout)')
```

Comment:
"Output BED12" reads a little unnatural. Do you think it would be ok to just specify these as BED? e.g. "Output BED to write" and `<BED>`